news/templates/news/list.html

```diff
@@ -55,7 +55,7 @@
 </table>
 <br />
 <p>
-	<a class="btn btn-primary btn-xs" href="{{ instance|url:'news-add' }}">
+	<a class="aplus-button--secondary aplus-button--sm" href="{{ instance|url:'news-add' }}">
```

Comment:
There are a few pages like this one that now have no primary actions. I was wondering why adding new items can't be the primary action. Is it so because the users may often come to this page for editing something and adding therefore isn't always the most used default action?

Reply:
I never meant to indicate that adding a new item can't be the default action, just that I didn't know that it should be. Pages can have "at most one" default action, so zero is ok too when there are several equal options (login methods for example). 

I don't know enough about how this page is used, but if you believe that most users landing on this page will want to create a new one, I'd be happy to swap it. If most will want to do something else (look at the list, edit or delete one), then maybe it makes more sense to not have a default action on this page. 

What do you think in this case?

Reply:
I see that drawing the line is sometimes blurry. That's OK. Regarding the news, I have the feeling that most teachers would usually add more news items and usually not modify existing news items.

Reply:
The news page now uses `aplus-button--default` for the "New" button as suggested.